docset/windows/printmanagement/Add-Printer.md

```diff
@@ -508,6 +508,9 @@ Accept wildcard characters: False
 Shows what would happen if the cmdlet runs.
 The cmdlet is not run.
 
+> [!NOTE]
+> The WhatIf switch doesn’t work if **ConnectionName** parameters set is used.
```

Comment:
```suggestion
The WhatIf switch does not work with this cmdlet.
```
In other repos I was suggested to add a simple sentence like this one in the cmdlets that WhatIf doesn't work. Also, Get-Help won't show correctly the note.

Reply:
But here it does not work only in a specific parameter set. For the other one it works, I have tested it.

Reply:
We could leave the current description proposed by @velkovb as he tested it is only happening for a specific param set but remove the note tag...?

Reply:
Agree, it's very strange.

Reply:
@dariomws Otherwise, I was looking at your PRs on the topic and almost had it committed when I did a quick test and found out it is different.

Reply:
@get-itips @dariomws What do you think about moving the notes in the NOTES section of the article?

Reply:
That could be a possibility... @dariomws ?

Reply:
@velkovb @get-itips like it! If it's something critical maybe it's not the best option, but in this case, I think it's perfect.